x-pack/plugins/triggers_actions_ui/public/application/hooks/use_bulk_edit_response.tsx

```diff
@@ -124,18 +125,22 @@ export function useBulkEditResponse(props: UseBulkEditResponseProps) {
       if (numberOfErrors === total) {
         toasts.addDanger({
           title: failureMessage(numberOfErrors, translationMap[property]),
-          text: toMountPoint(renderToastErrorBody(response), { i18n: i18nStart, theme }),
+          text: toMountPoint(renderToastErrorBody(response), {
+            i18n: i18nStart,
+            theme,
+            userProfile,
+          }),
         });
         return;
       }
 
       // Some failure
       toasts.addWarning({
         title: someSuccessMessage(numberOfSuccess, numberOfErrors, translationMap[property]),
-        text: toMountPoint(renderToastErrorBody(response), { i18n: i18nStart, theme }),
+        text: toMountPoint(renderToastErrorBody(response), { i18n: i18nStart, theme, userProfile }),
```

Comment:
Not passing a large object here for the services, because this is within a `useCallback`. See https://github.com/elastic/kibana/pull/202606#discussion_r1871588439